bus-mapping/src/evm/opcodes/sstore.rs

```diff
@@ -187,7 +196,7 @@ mod sstore_tests {
             .unwrap();
 
         assert_eq!(
```

Comment:
Before this we could check that the `CallContext` ops were pushed in the appropriate/expected order?

Reply:
updated in https://github.com/appliedzkp/zkevm-circuits/pull/467/commits/7e7470ad9b0a7a602e0348d70c81c65e59a57cc2 thanks